data/configs/default.json

```diff
@@ -10,7 +10,8 @@
     }
   ],
   "settings": {
-    "searchUrl": "https://google.com/search?q="
+    "searchUrl": "https://google.com/search?q=",
+    "searchNewTab": true
```

Comment:
It is if I want the switch to be toggled on by default. If there's no value there it defaults to false, so this seemed to make sense as the searchUrl is also set in the defaults.
I was having lots of issues with the boolean as if the value is null it can also be treated as false. Using ?? fixed the issue.